MAINTAINERS.md

```diff
@@ -0,0 +1,199 @@
+
+Here is [an example](https://github.com/openpgpjs/openpgpjs/issues/1661) of this occuring
+
+If you find yourself doing this a lot, your `.github/ISSUE_TEMPLATE` needs improvement. The template should be effectively guiding people in need of support to the discussions and away from the issues.
+
+If you realise that the question is a duplicate of an existing discussion, and you can easily find that discussion, it’s worth linking to the existing one instead of creating a new one.
+
+If questions are frequently asked, they might be a frequently asked question! Consider adding the solution to the documentation, Wiki or FAQ, whichever applies and is easily discoverable.
+
+#### `Wontfix` for misplaced or irrelevant issues
+
+These are issues that pertain to other projects but have landed here, usually because the poster has no insight into the source of the issue, and the source is actually a dependency.
+
+If you can, close the issue with a comment that points the poster to the correct repo.
+
+#### `Need more info` for incomplete issues
+
+These are issues that seem worthwhile, but are not yet actionable. Give it the `Need more info` label and prod the author for that info.
+
+#### `Stale` for abandoned issues
+
+These are usually `Need more info` issues where the original author does not respond within a sensible time frame (say, a month. People are busy). These are good candidates for a Github bot to manage for you.
+
+Here’s [an example](https://github.com/openpgpjs/openpgpjs/issues/883) of an issue that was manually closed after the author not responding for a year.
+
+#### `Bug` for things that don’t work as intended
+
+1. Assuming this is not or no longer a `Need more info` issue, attempt to reproduce the bug
+   1. Search for duplicates, if one exists, point the current issue there, and close the new one
+   2. If you can reproduce it and there are no duplicates, prioritize it and add the `Bug` label
+      1. If you need more info before this issue becomes actionable despite being able to reproduce it, prod the author and add a `Need more info` label
+   3. If you can’t reproduce it, contact the author with the reason, and add a `Need more info` to the `Bug` label
+      1. If the issue cannot be reproduced by anyone other than the author, agree with them that the issue should be closed
+
+As with regular `Need more info` issues, `Need more info` + `Bug` issues are also subject to the [Abandoned/Stale](#abandonedstale) rule above, and should be closed after a set period of time.
+
+#### Good First Issues and Help Wanted Issues
+
+To identify issues that are specifically groomed for new contributors, we use the `Help wanted` _and_ `Good first issue` labels. As to the difference between the two:
+
+- a `Good first issue` always also implies `Help wanted` from outside the project
+- but `Help wanted` does not necessarily mean it’s a `Good first issue`. Might be really tricky but none of the current maintainers has the resources to tackle it
```

Comment:
The first and last sentences seem contradictory to me... I would go with the latter interpretation, and drop the promise that "Help wanted" issues are suitable for new contributors - they should just mean that the core contributors won't get to it, IMO.